pkg/server/skydb/pq/listener.go

```diff
@@ -142,14 +142,15 @@ func (l *recordListener) Listen() {
 
 // NOTE(limouren): pending_notification.id is integer in database.
 func (l *recordListener) fetchNotification(notificationID string, n *notification) error {
+	logger := logging.Logger("push")
```

Comment:
This package should not be using the `push` logger. Does this package has a named logger? If not, create one in `skydb/pq/log.go` and use it in this package.